resources/views/config/main.blade.php

```diff
@@ -0,0 +1,66 @@
+@extends('layout')
+
+@section('head')
+    <link rel="stylesheet" href="{{asset('stylesheets/configStyle.css')}}">
+    <link rel="stylesheet" href="{{asset('stylesheets/mainGameScreenStyle.css')}}">
+    <script src="{{asset('scripts/keyGen.js')}}" defer></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
```

Comment:
Deze regel zou ik in `layout.blade.php` plaatsen, omdat je niet alleen voor één pagina jQuery wilt includen.

Reply:
Ik denk persoonlijk niet dat het nodig is door de geringe hoeveelheid schermen die er zullen komen in de webapp en het feit dat het belangrijkste scherm voor jQuery (het configuratiescherm zelf) nu gebruik maakt ervan

Reply:
Als het goed is gaan we toch maar een webpagina krijgen?